test/e2e/log_size_monitoring.go

```diff
@@ -212,9 +215,11 @@ func (g *LogSizeGatherer) Work() bool {
 	)
 	if err != nil {
 		Logf("Error while trying to SSH to %v, skipping probe. Error: %v", workItem.ip, err)
+		workItem.backoffMultiplier *= 2
```

Comment:
Doesn't really matter, as no test actually takes more than an hour, so getting to 64 means that we won't do it anymore.